test/test_indHyperslab.jl

```diff
@@ -1,13 +1,10 @@
 using LinearAlgebra
 using ProximalOperators
-using Random
 using Test
 
-Random.seed!(0)
-
 @testset "IndHyperslab" begin
 
-for R in [Float16, Float32, Float64]
+for R in [Float32, Float64]
```

Comment:
@nantonel I just removed Float16 here since one check we do is a bit tricky for indicator functions, especially with low precision